index.bs

```diff
@@ -369,33 +373,24 @@ spec: webidl
   </section>
 </section>
 <section>
-  <h2 dfn-type="dfn" export>
-    Permission Store
+  <h2 id="permission-stores">
+    Permission Stores
   </h2>
   <p>
-    <a lt='user agent'>User agents</a> MAY use a form of storage to
-    keep track of web site permissions. When they do, they MUST have a
-    <dfn export>permission storage identifier</dfn> which is linked to a
-    {{PermissionStorage}} instance or one of its subtypes.
+    A <dfn export>permission store</dfn> maps {{PermissionName}}s to instances
+    of {{PermissionStorage}} or one of its subtypes. A <a>permission store</a>
+    has one or more associated <a>origin</a>s, and SHOULD have only one.
   </p>
   <p>
-    To <dfn>get a permission storage identifier</dfn> for a
-    {{PermissionName}} <var>name</var> and an <a>environment settings
-    object</a> <var>settings</var>, the UA MUST return a tuple consisting
-    of:
+    The <a>user agent</a> MAY maintain any number of <a>permission store</a>s to
+    record what capabilities users have granted web sites permission to access.
+    The <a>user agent</a> may create new <a>permission store</a>s, for example
+    to serve a particular <a>origin</a>, <a>origin</a> within a <a>top-level
+    browsing context</a>, or <a>realm</a>, and when it does so, it may
+    initialize it using the mappings of any other <a>permission store</a>
+    associated with the <a>same origin</a>.
   </p>
```

Comment:
I did mean per-origin. I just meant that several top-level browsing contexts of the same origin would share the same permission. Though maybe not if some other setting was in play, like privacy mode, but we generally haven't defined or acknowledged privacy mode in standards.